data/packages.yml

```
@@ -61,4 +61,12 @@ forcats:
   url_docs: https://forcats.tidyverse.org/
   url_cran: https://cloud.r-project.org/web/packages/forcats/index.html
   url_hex: https://raw.githubusercontent.com/rstudio/hex-stickers/master/PNG/forcats.png
-  name: forcats
\ No newline at end of file
+  name: forcats
+lubridate:
+  weight: 80
+  desc: "lubridate makes it easier to do the things R does with date-times and possible to do the things R does not."
```

```suggestion
  desc: "lubridate provides a set of functions for working with date-times, extending and improving on R's existing support for them."
```

I know the existing description comes from the lubridate docs, but I find that sentence quite confusing and feel like we have an opportunity to make it a little more straightforward here

I also like that stringr, forcats, and lubridate all start with `"<pkg> provides..."`